src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -26,11 +26,11 @@
     private static final Logger log = Logger.getLogger();
     private static final String STATISTICS_NO_VALUE_STRING = "-";
 
-    private boolean hasAssignedWeights;
-    private List<Double> rubricWeights;
-    private int numOfRubricChoices;
+    boolean hasAssignedWeights;
```

Comment:
Understand that this is to avoid `AccessorMethodGeneration`, but will this violate encapsulation? Now all the question details classes get to see these variables.